src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1051,13 +1052,17 @@ private static synchronized Locale getFormatLocale() {
         return loc;
     }
 
+    @SuppressWarnings("removal")
     private static Locale initDefault() {
         String language, region, script, country, variant;
-        Properties props = GetPropertyAction.privilegedGetProperties();
-        language = props.getProperty("user.language", "en");
+        var sm = System.getSecurityManager();
+        if (sm != null) {
+            sm.checkPropertiesAccess();
+        }
```

Comment:
This SM check is no longer needed; the use of privilegedGetProperties made access unconditional.
The static properties are always accessible to the locale implementation.

Reply:
Was wondering so. Will remove them.